app/controllers/media_controller.rb

```diff
@@ -39,4 +39,9 @@ def destroy
       process_error
     end
   end
+
+  def clone
+    @medium = Medium.find(params[:id]).dup
```

Comment:
You can add `:clone` to `before_action` and then just do `@medium.dup`

Reply:
Doing so results in a following error: ERF42-5531 [Foreman::Exception]: unknown permission for media#clone
Should I change the permissions (in that case I'll need a hint where to find them) or leave it be?